blockchain/fullblocktests/generate.go

```diff
@@ -1462,17 +1554,75 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	//                 \-> b51(15)
 	g.SetTip("b46")
 	g.NextBlock("b51", outs[15], ticketOuts[15], func(b *wire.MsgBlock) {
+		// set the merkle root to an invalid hash
 		b.Header.MerkleRoot = chainhash.Hash{}
 	})
 	g.AssertTipBlockMerkleRoot(chainhash.Hash{})
 	rejected(blockchain.ErrBadMerkleRoot)
 
-	// Create block with an invalid proof-of-work limit.
+	// Create block with an invalid stake root
 	//
 	//   ... -> b46(14)
 	//                 \-> b52(15)
 	g.SetTip("b46")
 	g.NextBlock("b52", outs[15], ticketOuts[15], func(b *wire.MsgBlock) {
```

Comment:
`b52` can't be changed like this or it will cause the `b52a` test to not test the intended thing.  More generally, you can't reuse any of the same names.  I think I'll add an assertion to `NextBlock` to causes duplicates to fail.